packages/api/src/graphql/query.public.ts

```diff
@@ -63,6 +64,7 @@ export const GraphQLPublicQuery = new GraphQLObjectType<undefined, Context>({
     peer: {
       type: GraphQLPeer,
       args: {id: {type: GraphQLID}, slug: {type: GraphQLSlug}},
+      description: 'This query takes either the ID or the slug and returns the peer profile.',
```

Comment:
Just a note that you sometimes write ID lowercase, and probably best to be consistent.